src/anemoi/datasets/create/input/result.py

```diff
@@ -30,6 +30,19 @@
 LOG = logging.getLogger(__name__)
 
 
+def _fields_metatata(variables, cube, namespace="mars"):
```

Comment:
Am I right, this builds the metadata given a namespace? Could you add a brief description?